src/Codeception/Module/Facebook.php

```diff
@@ -121,11 +123,17 @@ public function _requires()
 
     public function _depends()
     {
-        return ['Codeception\Module\PhpBrowser' => $this->dependencyMessage];
+        return ['Codeception\Lib\Interfaces\Remote|Codeception\Lib\Interfaces\Remote' => $this->dependencyMessage];
```

Comment:
I think it won't work this way :(
Our parser is not so smart to handle both interfaces... Especially two the same interface...
Let's ask for `Codeception\Lib\Interfaces\Web` interface here